### DISCUSSION TITLE
Improve sticky sidebar

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#265] [Ideas]
I noticed that the sticky sidebar is partly hidden when reaching the end of the website, even when there is space to show it fully.

Examples:

**Full sidebar**

![PushEVs_sidebar1](https://github.com/hbstack/theme-cards/assets/49299328/3a47c530-4cea-451d-b035-da98e5cd9e86)

**Sidebar partly hidden when reaching the bottom**

![PushEVs_sidebar2](https://github.com/hbstack/theme-cards/assets/49299328/a8e71fb8-f84f-4826-86f8-73cc75ecf192)

**How it should look**

![PushEVs_sidebar3](https://github.com/hbstack/theme-cards/assets/49299328/e257aa7a-1e28-4a75-b7f7-3dfb61e50e68)

How can we configure the sidebar to scroll down til the footer element?

I wonder if this be can be solved with CSS/bootstrap or it needs JS like [this](https://abouolia.github.io/sticky-sidebar/) old project seems to suggest.

**Comment by razonyang** [comment]
Duplicate to #131, let's keep it open to see how many users like to _improve_.

**Reply by LITUATUI** [reply]
Can you please reopen that issue? The sidebar on hbstack.dev would be OK.

**Comment by razonyang** [comment]
Closing now, please track this on #131.
